semantic_model_generator/validate_model.py

```diff
@@ -41,18 +43,21 @@ def validate(yaml_str: str, snowflake_account: str) -> None:
         account_name=snowflake_account,
         max_workers=1,
     )
+    model_in_column_format = context_to_column_format(model)
 
-    for table in model.tables:
+    for table in model_in_column_format.tables:
         logger.info(f"Checking logical table: {table.name}")
         # Each table can be a different database/schema.
         # Create new connection for each one.
         with connector.connect(
             db_name=table.base_table.database, schema_name=table.base_table.schema
         ) as conn:
             try:
-                select = generate_select_with_all_cols(table, 100)
-                # Run the query
-                _ = conn.cursor().execute(select)
+                sqls = [generate_select(table, 1)] + generate_agg_expr_selects(table, 1)
+                # Run the query.
+                # TODO: some expr maybe expensive if contains aggregations or window functions. Move to EXPLAIN?
```

Comment:
+1, can we do this now?

Reply:
i am a bit unsure if explain captures all; let's discuss further before making this change